docs/releases/5.1.txt

```diff
@@ -153,7 +153,7 @@ Internationalization
 Logging
 ~~~~~~~
 
-* ...
+* `response_time` added in log messages generated by runserver.
```

Comment:
```suggestion
* ``response_time`` added in log messages generated by runserver.
```

Reply:
Or maybe "``runserver`` log messages now also include the response time."
Might be nice to make runserver a ref that links to the docs 🤔 
Make sure to run `make html` in the docs path and check any links work

Reply:
Updated, thanks :D